concordium-consensus/src/Concordium/GlobalState/Persistent/Instances.hs

```diff
@@ -119,10 +119,10 @@ instance MonadBlobStore m => BlobStorable m PersistentInstance where
 
 instance MonadBlobStore m => Cacheable m PersistentInstance where
     cache p@PersistentInstance{..} = do
-        -- TODO: We do not currently cache the pinstanceCachedParameters.
-        -- This behaviour is probably fine.
+        -- we only cache parameters and the interface. The rest is already in memory at this point.
```

Comment:
For the unenlightened ones it could be beneficial to add a small comment saying why only these 
two fields of the contract are required to be cached.

Reply:
I attempted to clarify the comment.